drivers/flight_gui.py

```diff
@@ -20,11 +20,22 @@ def vector_Plot(x, y, name=None, *args, **kwd_args):
     return plot.Plot(x, [d + "_" + y for d in ['x', 'y', 'z']], name, *args, **kwd_args)
 
 dts = ([manager.DataType('temperature', float, units='deg C', thresholds=(-20, 80))] +
-       vector_DataType('magnetometer', float, thresholds=(-100, 100)) + # TODO: units
-       vector_DataType('gyro', float, thresholds=(-100, 100)) + # TODO: units
+       [manager.DataType('bmp_alt', float, units='m', thresholds=(-100, 80000))] +
```

Comment:
Style nit: these can all be one list instead of appending a bunch of single-element lists.  i.e.
```python
[manager.DataType('bmp_alt', float, units='m', thresholds=(-100, 80000)),
 manager.DataType('gps_alt', float, units='m', thresholds=(-100, 80000)),
 ...
 manager.DataType('sats', float, units='#', thresholds=(-10, 169))]
```